### DISCUSSION TITLE
Debugging <link rel="me">

### DISCUSSION THREAD

**Discussion by JD557** [mastodon/mastodon#22381] [Q&A]
Hey everyone, I would like some help to debug the link rel="me" integration. Is there any tool that I can use?

I'm actually not using mastodon, so this makes the normal debug process quite tricky (as I need to wait for my profile to sync with another instance and see if I get the URL verified)

For reference, a few things that I suspect might make it not work for me:
- I'm running a single instance blog, so my profile URL is just the blog URL (no `@JD557` in the URL)
- My website replies with a 302 redirect

**Comment by mjankowski** [comment]
I realize this is three years later and no one has responded, but ... did you ever work this out?

If not, can you post the URL in question and I'll have an opinion for you.

**Reply by JD557** [reply]
Hey, thanks.
I'm on holidays without a computer, so I might be slow to respond, but here are the details:

The website in question is: https://www.joaocosta.eu/

Which redirects to https://www.joaocosta.eu/Portfolio (and includes the link rel)

The instance is https://blog.joaocosta.eu (although I made some changes and https://blog.joaocosta.eu/@JD557 should also work)

**Reply by mjankowski** [reply]
Thanks for all that ... the markup all looks fine, I think. Typically the verification would happen by doing a `rel=me` and pointing to a Mastodon profile URL, which would see that pointing at itself and call it verified.

It's not clear to me here which thing you are expecting to do the verification, though... ?

**Reply by JD557** [reply]
I was expecting to see a verification mark on my website entry when people check my profile via mastodon.

![Screenshot_20251120-161705.png](https://github.com/user-attachments/assets/99c00422-0a70-4064-9fe4-276b1f6aceee)

Unless this feature is just for mastodon profiles, not all ActivityPub profiles.

**Reply by mjankowski** [reply]
Ah ... so you're trying to see how you can you get your (non Mastodon) AP profile published collection of links to show up in other people's mastodon servers when they've interacted with the profile?

**Reply by JD557** [reply]
Exactly 
The links show, but I can't get the verification mark.
